ci/ansible/roles/pulp-backup/tasks/main.yaml

```diff
@@ -0,0 +1,52 @@
+# rhel6 pulp 2.11.z backup
+- name: Stop services before backup
+  service: "name={{ item }} state=stopped"
+  with_items:
+    - httpd
+    - pulp_celerybeat
+    - pulp_resource_manager
+    - pulp_streamer
+    - pulp_workers
+    - qpidd
+  notify: Restart services
+
+- block:
```

Comment:
I think it is an artifact of when I originally had this as a block you could skip in the back up step. I'll remove it